samples/webview-samples/multi-tab-browser/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "manifest_version": 2,
+  "name": "Multi-tabbed Browser",
+  "minimum_chrome_version": "24.0.1307.0",
```

Comment:
`webview.setUserAgentOverride` is at least Chrome 33. Please make sure we don't use other methods that require more than Chrome 33.